docs/sdks/survey-sdk/responses-polling.md

```diff
@@ -0,0 +1,95 @@
+
+Engage Digital will poll your survey responses by performing a `GET /responses` on your bridge every 15 minutes along with a `since` parameter corresponding to the the most recent response fetched's `submitted_at` field.
+
+This parameter will be sent as milliseconds in UTC time (e.g. if **base URL of the bridge** is `https://survey.bridge.com` and your most recent response was submitted on `2022-01-10T10:00:00.000Z` then Engage Digital will perform a `GET https://survey.bridge.com/responses?since=1641808800000`).
+
+## Response example
+
+Here's an example of a valid response to the responses polling:
+```json
+{
+  "id": "12345",
+  "records": [
+    {
+      "id": "12345_response_1",
+      "submitted_at": "2022-01-10T10:00:00.000Z",
+      "mapping_key": "response_mapping_key",
+      "questions": [
+        {
+          "id": "main_question",
+          "replies": [
+            {
+              "value": "5"
+            }
+          ]
+        },
+        {
+          "id": "multiple_choices_question",
+          "replies": [
+            {
+              "value": "Faster response time"
+            },
+            {
+              "value": "Better follow-up"
+            }
+        ]
+        },
+        {
+          "id": "free_text_question",
+          "replies": [
+            {
+              "value": "Everything's already perfect"
+            }
+          ]
+        },
+        ...
+      ]
+    },
+    ...
+  ]
+}
+```
+
+
+### Detailed response format
+
+| Field | Type | Mandatory | Description |
+|-|-|-|-|
+| id | String | NO | Unique identifier of the survey. |
+| records | Array | **YES** | List of responses, see [Response object description](#response-object-description). |
+
+
+### Response object description
+
+| Field | Type | Mandatory | Description |
+|-|-|-|-|
+| id | String | **YES** | Unique identifier of the response. |
+| submitted_at | Time | **YES** | Time the response was submitted at, the entire response will be ignored if not present. |
+| mapping_key | String | **YES** | Mapping key that was sent as the `i` parameter in the survey link, see [Response mapping key](#response-mapping-key). |
+| questions | Array | **YES** | List of questions and their associated replies, see [Question object description](#question-object-description). |
+
+
+### Question object description
+
+| Field | Type | Mandatory | Description |
+|-|-|-|-|
+| id | String | **YES** | Unique identifier of the question, the whole entry will be ignored if not present. |
+| replies | Array | **YES** | List of replies for a given question (can have multiple entries), see [Reply object description](#reply-object-description). |
+
+
+### Reply object description
+
+| Field | Type | Mandatory | Description |
+|-|-|-|-|
+| value | String | **YES** | Value of the reply (in the [example above](#response-example), value 5 will be equivalent to "Very satisfied" as seen in the [configuration](../configuration-fetching#response-example)). |
```

Comment:
> So if there is no value here in the configuration, and the text is "Very satisfied", and on the response polling, we need to have the value to "Very satisfied" too, right? If so, that's what needs to be more explicit

We should improve this description now. Something like this?

```suggestion
| value | String | **YES** | Value of the reply (in the [example above](#response-example), value "5" will be equivalent to "Very satisfied" as seen in the [configuration](../configuration-fetching#response-example). If there is no value in the configuration, this value will need to be the text one, e.g. "Very satisfied"). |
```

Reply:
actually it's not even mandatory, we're storing the reply's value regardless of the configuration and when we need to display the reply we're checking if there's a text matching the value in the config otherwise we simply display the value

so basically the choice configuration is only useful when a value <-> text mapping is needed but otherwise it's behaving just like a free text question and display whatever the reply value is

Reply:
ah ok cool